packages/samples/headless-ssr/src/app/common/result-list.tsx

```diff
@@ -0,0 +1,17 @@
+import {Result} from '@coveo/headless';
+
+interface ResultListCommonProps {
+  results: Result[];
```

Comment:
I think this would be more copy and paste-able (and possibly easier to read as a sample) if we changed the props of common components to match the values returned by the hook (which also happen to be the props I used in the generic ssr samples).

For example:
```suggestion
  state: ResultListState;
  methods: ResultListMethods;
```